expenses-tracker-frontend/.env.example

```diff
@@ -0,0 +1,2 @@
+VITE_GOOGLE_LOGIN_CLIENT_ID=
+VITE_SERVER_BASE_URL=http://localhost:3000
```

Comment:
FYI: this will conflict with the BE when you run both of the apps, because the BE also runs on port 3000

Reply:
This is supposed to be the BE's address, I've added it so in the future we can send the requests to that address